.github/workflows/run-kata-coco-tests.yaml

```diff
@@ -42,7 +42,9 @@ jobs:
       KATA_HYPERVISOR: ${{ matrix.vmm }}
       KUBERNETES: "k3s"
       USING_NFD: "true"
+      KBS: "true"
       K8S_TEST_HOST_TYPE: "baremetal"
+      KBS_INGRESS: "nodeport"
```

Comment:
@wainersm is this KBS Ingress correct or should I use the aks?

Reply:
Hi @GabyCT ! Yes, on TDX machine we've k3s so it should use the nodeport ingress.